src/plugins/intel_gpu/include/intel_gpu/runtime/format.hpp

```diff
@@ -83,6 +83,9 @@ struct format {
         byxf,                                   ///< used in bitmaps, input from user i.e b images of RGB format
         fbyx,
         fyxb,                                   ///< format not used inside clDNN, but supported in reorder as extension
+        fybx,                                   ///< To be used when onednn gemm allows permute fusing in transformer network. Not for normal use from cldnn.
+        xbfy,                                   ///< To be used when onednn gemm allows permute fusing in transformer network. Not for normal use from cldnn.
+        ybfx,                                   ///< To be used when onednn gemm allows permute fusing in transformer network. Not for normal use from cldnn.
```

Comment:
I wonder if we really need these new tags? I'd expect that permute fusion can be done in a similar way to ocl kernels case, but when we create memory desc we can adjust it using in/out transpose orders

Reply:
Yes I have that task in my backlog, will proceed it after the CF :)